src/test/java/fi/aalto/cs/apluscourses/model/CourseTest.java

```diff
@@ -163,12 +171,13 @@ public void testGetModuleWithMissingModule() throws NoSuchComponentException {
   private static String autoInstallJson = "\"autoInstall\":[\"O1Library\"]";
   private static String replInitialCommands = "\"repl\": {\"initialCommands\": {\"GoodStuff\": ["
       + "\"import o1._\",\"import o1.goodstuff._\"]}}";
+  private static String courseVersion = "\"version\": \"5.8\"";
 
   @Test
   public void testFromConfigurationFile() throws MalformedCourseConfigurationException {
     StringReader stringReader = new StringReader("{" + idJson + "," + nameJson + "," + urlJson
         + "," + languagesJson + "," + modulesJson + "," + exerciseModulesJson + "," + resourcesJson
-        + "," + autoInstallJson + "," + replInitialCommands + "}");
+        + "," + autoInstallJson + "," + replInitialCommands + "," + courseVersion + "}");
```

Comment:
This is perhaps more of an issue with how the tests are implemented, but now all of the malformed JSON tests that are later in this file don't actually test what they claim to test; all of them would fail in any case because the missing version causes an exception.